app/services/marketing_site.rb

```diff
@@ -1,6 +1,10 @@
 class MarketingSite
   BASE_URL = URI('https://login.gov').freeze
 
+  def self.base_url
+    BASE_URL
```

Comment:
anything in a view will have `to_s` called on it by default (HELLO RAILS ✨ ). 

I think you know this and so you are suggesting that we call `to_s` here so we  know in our tests we are testing the correct behavior? Want to make sure I understand